docs/pages/example/fill-pattern.js

```diff
@@ -0,0 +1,10 @@
+/*---
+title: Add a pattern to a polygon
+description: Load an image and use it as a repeating pattern to draw a polygon.
```

Comment:
The description should include the term `fill-pattern` so that if the property is searched in the examples, this example will come up. Something like `Use [fill-pattern](https://docs.mapbox.com/mapbox-gl-js/style-spec/#paint-fill-fill-pattern) to draw a polygon from a repeating image pattern` would be good and I like linking to the property documentation.